draft-packages/table/KaizenDraft/Table/Table.tsx

```diff
@@ -91,6 +93,14 @@ export const TableHeaderRowCell: TableHeaderRowCell = ({
   onCheck,
   active,
   automationId,
+  // I can't say for cetin why "nowrap" was the default value. Normally you wouldn't
+  // want to clip off information because it doesn't fit on one line.
+  // My assumption is that because since the cell width rows are decoupled, a heading
+  // cell with a word longer than the column width would push the columns out of
+  // alignment? I'm not sure.
+  // Anyway, we can override this default behaviour by setting wrapping to "wrap".
+  wrapping = "nowrap",
```

Comment:
This is to preserve existing behaviour. Would it make sense to either turn wrapping on by default, or turn on an ellipsis instead of using `nowrap`?

Reply:
> I can't say for cetin why "nowrap" was the default value.
I'd hazard a guess it's because of this:

```
Responsive:

    On small screens, the table is sideways scrollable.
    On touch devices (small or large), scroll is available on touch.
```
https://cultureamp.design/components/table/

But I could be wrong ¯\_(ツ)_/¯

Reply:
If that's the case, would you still think it make sense to turn wrapping on by default? You can still have wrapped header cells along with a table that requires horizontal scrolling.

Reply:
It's probably safer to leave as is.